lea/clients/bigquery.py

```diff
@@ -98,7 +99,7 @@ def delete_view_key(self, view_key: tuple[str]):
         self.client.delete_table(f"{self.project_id}.{table_reference}")
 
     def _read_sql_view(self, view: lea.views.View) -> pd.DataFrame:
-        return pd.read_gbq(view.query, credentials=self.client._credentials)
+        return read_gbq(view.query, credentials=self.client._credentials, progress_bar_type=None)
```

Comment:
```suggestion
        return pandas_gbq.read_gbq(view.query, credentials=self.client._credentials, progress_bar_type=None)
```

Can you do this instead? When using external libraries, I think it's more Pythonic to include the module name